### ISSUE TITLE
backface-visibility needs -webkit- prefix for Safari OS X and iOS

### ISSUE THREAD

**Issue by PascalPencil** [postcss/autoprefixer#653]
backface-visibility seems to need -webkit- prefix for Safari OS X and iOS
please check the 3D flip cards halfway down the page here http://www.superpencil.com/en/work/fishisfast to confirm

**Comment by ai** [comment]
Nope http://caniuse.com/#feat=transforms3d

**Comment by PascalPencil** [comment]
Yep:

Line on caniuse repo here: https://github.com/Fyrd/caniuse/blob/master/features-json/transforms3d.json#L281

<img width="363" alt="screenshot 2016-05-13 22 05 33" src="https://cloud.githubusercontent.com/assets/3094044/15248782/f8368c38-1956-11e6-8d51-2c5d611cad5a.png">
<img width="363" alt="screenshot 2016-05-13 22 05 46" src="https://cloud.githubusercontent.com/assets/3094044/15248783/fa421a6a-1956-11e6-9347-9451be71bbcd.png">

You can manually patch it for now or just wait for them to, meanwhile autoprefixer doesn't work for this.

**Comment by ai** [comment]
Ouh :( Thanks for examples and link

**Comment by thetalecrafter** [comment]
not only does it fail to add the prefix, but it removes the prefixed property if you manually add it. :(

**Comment by ai** [comment]
@thetalecrafter just add some old Safari as temporary solution. I will be able to fix it only few days later.

**Comment by thetalecrafter** [comment]
@ai Thanks for the workaround!

**Comment by jkneb** [comment]
Kinda same problem with flexbox on Safari iOS. I lost all the `display:-webkit-flex` …

Adding 'iOS 7' in my supported browsers solved the issue.

``` js
autoprefixer({ 
  browsers: ['last 2 versions', 'iOS 7'] 
})
```

**Comment by ai** [comment]
@jkneb your problem could be fixed by `browserslist` config https://github.com/postcss/autoprefixer#no-prefixes-in-production

**Comment by ai** [comment]
Done 4e27060

Will be released in 6.4.

**Comment by ai** [comment]
Released in 6.4.
